README.md

```diff
@@ -3,7 +3,7 @@
 The recommended way to start a Vite-powered Vue project
 
 <p align="center">
-  <img src="https://github.com/vuejs/create-vue/blob/main/media/screenshot-cli.png?raw=true" width="802">
+  <img width="898" alt="Screenshot 2024-11-10 at 20 58 46" src="https://github.com/user-attachments/assets/ca4a908d-d98c-43a8-bb86-ba0ba9ba7eb1">
```

Comment:
I think the alt should be more descriptive.

But I'm mostly wondering why the png from the repo is not updated instead of referencing an external asset?